src/Figure/Triangle.java

```diff
@@ -0,0 +1,45 @@
+package Figure;
+
+public class Triangle implements Area {
+    private double a; // довжина першой сторони
+    private double b; // довжина друйго сторони
+    private double c; // довжина третьой сторони
+
+    public double getA() {
+        return a;
+    }
+
+    public void setA(double a) {
+        this.a = a;
+    }
+
+    public double getB() {
+        return b;
+    }
+
+    public void setB(double b) {
+        this.b = b;
+    }
+
+    public double getC() {
+        return c;
+    }
+
+    public void setC(double c) {
+        this.c = c;
+    }
+
+    public Triangle(double a, double b, double c) {
```

Comment:
Do not forget about the class members ordering: constructors should be declared right after the instance variables.